test/loggers/WinstonLoggyslavTest.ts

```diff
@@ -66,4 +70,53 @@ export class WinstonLoggyslavTest {
 
         expect(firstCallArgs).deep.include([5, 3]);
     }
+
+    @test
+    private "Should call WinstonLogger when initialized as error logger"() {
+        const winstonNewLogger = new winston.Logger( {
+            level: "info",
+            transports: [
+                new winston.transports.Console(),
+            ],
+        } );
+        const winstonNewErrorLogger = new winston.Logger( {
+            level: "error",
+            transports: [
+                new winston.transports.Console(),
+            ],
+        } );
+        const winstonLogger = new WinstonLoggyslav(winstonNewLogger);
+        const winstonErrorLogger = new WinstonErrorLoggyslav(winstonNewErrorLogger);
+
+        const targetsConfiguration: TargetsConfiguration = {
+            targets: [
+                {
+                    classType: SimpleClass,
+                    methods: [
+                        "throwError",
+                    ],
+                },
+            ],
+        };
+        const loggerConfiguration: LoggerConfiguration = {
+            methodLogger: winstonLogger,
+            errorLogger: winstonErrorLogger,
+        };
+
+        this.initNewLogger(targetsConfiguration, loggerConfiguration);
+
+        const simpleClass = new SimpleClass();
+        const spy = sinon.spy(winstonErrorLogger, "error");
```

Comment:
When inside a sandbox, use `sandbox.spy` on your sandbox instance, instead of `sinon.spy`, otherwise the methods you spied won't be restored with `sandbox.restore()`